### ISSUE TITLE
Coupons Exception

### ISSUE THREAD

**Issue by jvasa** [smkent/safeway-coupons#11]
Getting this error:
Exception getting preferred store: 'definitions'
Traceback (most recent call last):
  File "/usr/local/safeway-coupons/safeway-coupons", line 258, in _clip_coupons
    if len(storeid_data['definitions'][0]['values']) > 0:
KeyError: 'definitions'

It doesn't seem to affect the coupon clipping.

**Comment by smkent** [comment]
Thanks for the report! This error is happening when trying to retrieve the Your Club Specials (YCS) list, which is now returning 403 Forbidden.

I can't find the Your Club Specials feature on Safeway's site anymore, so I presume it has been removed. I have removed all the YCS-related code in 095f7c2ca07d77ad50dee1239fbc123c4c081b38 which should silence this error. YCS entries weren't coupons and didn't need to be clipped in order to work in the first place; it appeared to just be a shopping list feature.
